tests/PHPUnit/Util/Log/VCRTest.php

```diff
@@ -23,4 +23,22 @@ public function testInterceptsWithAnnotations()
         $result = file_get_contents('http://google.com');
         $this->assertEquals('This is a annotation test dummy.', $result, 'Call was not intercepted (using annotations).');
     }
+
+    /**
+     * @vcr unittest_annotation_test
+     * @dataProvider aDataProvider
+     */
+    public function testInterceptsWithAnnotationsWhenUsingDataProvider($dummyValue)
+    {
+        // Content of tests/fixtures/unittest_annotation_test: "This is a annotation test dummy".
+        $result = file_get_contents('http://google.com');
+        $this->assertEquals('This is a annotation test dummy.', $result, 'Call was not intercepted (using annotations with data provider).');
+    }
+
+    public function aDataProvider()
+    {
+        return array(
+            array(null)
```

Comment:
maybe you can put another item, this way we can make sure multiple iterations are happening. What do you think?